packages/@uppy/companion/src/standalone/helper.js

```diff
@@ -51,7 +51,10 @@ const getConfigFromEnv = () => {
       },
       zoom: {
         key: process.env.COMPANION_ZOOM_KEY,
-        secret: getSecret('COMPANION_ZOOM_SECRET')
+        secret: getSecret('COMPANION_ZOOM_SECRET'),
+        custom: {
+          verificationToken: getSecret('COMPANION_ZOOM_VERIFICATION_TOKEN')
```

Comment:
`verificationToken` is a required option, right? can we put it on the `zoom` options object itself instead of inside `custom`?

Reply:
it might need a better name from `custom`, I agree.

I added this because the providers and their options are sometimes passed around and managed from a higher level that isn't aware of the specifics (e.g [here](https://github.com/transloadit/uppy/blob/master/packages/%40uppy/companion/src/server/provider/index.js#L114-L157)), so I thought it made sense to have a way to reference all of their provider specific options if the need arises.

So if for any reason we need to pass it somewhere or reference them somewhere, we can do `compainonOptions[providerName].custom`.  But I agree that `custom` may not be the best name.

Reply:
maybe rename it to `providerSpecific`? Or `internal`? What do you think?

Reply:
I think I still don't understand what the nesting does here. What kind of need could arise where we need to pass around provider-specific options, but not the general options? If that does happen, would `const { secret, key, ...specificOptions } = providerOptions` not work?

Reply:
ah right, `const { secret, key, ...specificOptions } = providerOptions` does work (didn't think of this). I am just trying to be careful, to provide options structure that won't need any breaking change in the future. But I didn't think about object destructuring this way at the time. I think it should do the trick, I'll go ahead and remove it.